sw/device/lib/testing/entropy_testutils.c

```diff
@@ -65,15 +82,32 @@ void entropy_testutils_auto_mode_init(void) {
   const dif_edn_auto_params_t edn1_params = {
       // EDN1 provides highest-quality entropy.  Let one generate command
       // return 1 block, and reseed after every generate.
-      .reseed_material =
+      .instantiate_cmd =
+          {
+              .cmd = 0x00000001 |  // Reseed from entropy source only.
+                     kMultiBitBool4False << 8,
+              .seed_material =
+                  {
+                      .len = 0,
+                  },
+          },
+      .reseed_cmd =
           {
-              .len = 1,
-              .data = {0x00000002 |  // Reseed from entropy source only.
-                       kMultiBitBool4False << 8},
+              .cmd = 0x00001002 |  // One generate returns 1 block, reseed
```

Comment:
Out of curiosity, where can I find the documentation for these commands?

Reply:
Here you go: https://docs.opentitan.org/hw/ip/csrng/doc/#command-header

Thanks for approving this :-)